### ISSUE TITLE
Patching to multiple VST's with Carla_2.0-RC1-win64 in PatchBay Mode

### ISSUE THREAD

**Issue by bencrossman** [falkTX/Carla#762]
So I have one midi input all on midi channel 1.
I want to write a plugin that will split this up and send to different VST's but their input is all still midi channel 1.

So the easy way is for my plugin in to split to different midi channels, each route filters for one of these channels and then will force it back to midi channel 1 before entering the VST.

Here is what I'm thinking of having (minus the actual plugin):

![multichannel](https://user-images.githubusercontent.com/13842596/46445536-bd629880-c7ba-11e8-80ff-90cf032c364e.png)

However I'd rather have a plugin that has multiple outs that I can connect straight to the VST.
I've had a play with internal plugins like Midi Split but get this:

![image](https://user-images.githubusercontent.com/13842596/46445473-68268700-c7ba-11e8-8b5e-a2abe91c2459.png)

I also read somewhere about JUCE only allowing one midi input and one midi output?
Is there a different plugin type I can use?

Anyway is there a way I can do this or is the way I'm already doing it the best way.
Cheers
Ben

**Comment by bencrossman** [comment]
Yeah that's how I've been doing it on Linux but it's Windows I'm trying to do similar.

**Comment by falkTX** [comment]
Yeah, there is no better way to do this yet.
The multiple MIDI ports feature is not that simple to implement. Might be something for v2.2
